pages/accountLists/[accountListId]/reports/designation_accounts.page copy 4.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactElement } from 'react';
```

Comment:
I think maybe this file name won't work.

Reply:
Oops, I forgot to finish renaming it after copying the file.